app/controllers/employees_controller.rb

```diff
@@ -1,6 +1,6 @@
 class EmployeesController < ApplicationController
   def index
-    @employees = Employee.all
+    @employees = Employee.all.where(on_vacation: false)
```

Comment:
Let's remember to ask about this. I think this action shouldn't be in the controller. I have it located in a view which I'm not sure about either.